modules/ROOT/pages/snippets/add_channels_scc_multiarch.adoc

```diff
@@ -1,5 +1,6 @@
 .Procedure: Adding Software Channels at the Command Prompt
-. At the command prompt on the {productname} Server, as root, use the [command]``spacewalk-common-channels`` command to add the appropriate channels:
+. At the command prompt on the {productname} Server, as root, use the [command]``spacewalk-common-channels`` command to add the appropriate channels.
```

Comment:
If this is about adding channels on scc multiarch, why do we mention `spacewalk-common-channels`?

Reply:
I'm using scc here to mean `spacewalk-common-channels` (the SUSE Customer Center method is called `webui`).

Reply:
Ah, now I see, but please pick other name. As you can it generates confusion.

And calling SUSE Customer Center method "webui" doesn't make sense, sorry. You can can also use CLI for SCC, even if you don't use `spacewalk-common-channels` but `mgr-sync`.

Check `mgr-sync` and you will see it also allows adding SUSE Customer Center channels without using the WebUI. I explained on the capture you mentioned `mgr-sync or WebUI`

Reply:
In the docs, we use either the WebUI Wizard, or we use mgr-sync, we don't refer to this as "SCC" in any context.

Reply:
@Loquacity please do not use "scc" for spacewalk-common-channels, not even in filenames. "SCC" is a well-known acronym for SUSE customers and we do not want to create any confusion at all, I do not even want to scratch that possibility.

Reply:
hrm, ok, what if we go with `swcc`?

Reply:
Personally I'd suggest `vendor` when we want to refer to channels from SCC/mgr-sync and `novendor` when we want to refer to channels from `spacewalk-common-channels`, and then you can use it in the future for snippets for vendor vs novendor

Reply:
I really like that idea, thanks for the suggestion!